src/libs/getPlaidLinkTokenParameters/index.js

```diff
@@ -0,0 +1,8 @@
+import ROUTES from '../../ROUTES';
+import * as Url from '../Url';
+import CONST from '../../CONST';
+
+export default () => {
+    const bankAccountRoute = window.location.href.includes('personal') ? ROUTES.BANK_ACCOUNT_PERSONAL : Url.removeTrailingForwardSlash(ROUTES.getBankAccountRoute());
```

Comment:
NAB, not sure we need a new method to `removeTrailingForwardSlash` for this? 
We could also create a new constant for `bank-account` and use that. I would recommend changing the existing `ROUTES.BANK_ACCOUNT` to something like `BANK_ACCOUNT_WITH_STEP_TO_OPEN`

Reply:
got it